docker/docker.go

```diff
@@ -161,6 +161,12 @@ func (d *ImageManager) FindImage(imageName string) (*dockerclient.Image, error)
 	return image, nil
 }
 
+// HasImage determines if the given image already exists in Docker
+func (d *ImageManager) HasImage(imageName string) bool {
+	_, err := d.FindImage(imageName)
+	return (err == nil)
```

Comment:
Would having it return `(bool, error)` and specifically checking for `docker.ErrImageNotFound` be okay?

(This was added so we can shim `FindImage()` without having to pull in the `docker` package.)